vcbuild.bat

```diff
@@ -83,6 +85,10 @@ if /i "%1"=="test-async-hooks"  set test_args=%test_args% async-hooks&goto arg-o
 if /i "%1"=="test-all"      set test_args=%test_args% gc internet pummel %common_test_suites%&set build_testgc_addon=1&set cpplint=1&set jslint=1&goto arg-ok
 if /i "%1"=="test-node-inspect" set test_node_inspect=1&goto arg-ok
 if /i "%1"=="test-check-deopts" set test_check_deopts=1&goto arg-ok
+if /i "%1"=="test-v8"       set test_v8=1&set custom_v8_test=1&goto arg-ok
```

Comment:
There are other checks that are not aligned. e.g. `test-all` above. `test-v8` aligns to `test-all` and other shorter if checks.